src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -0,0 +1,348 @@
+{% endAside %}
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/YNM3CfMKqnLuCJLw3I0U.png", alt="Oculus Browser inviting the user in a prompt to install the MyEmail app.", width="512", height="343" %}
+
+### App submission
+
+Apart from
+[submitting apps to the Oculus Store](https://developer.oculus.com/documentation/web/pwa-submit-app/),
+developers can also
+[submit their apps to App Lab](https://developer.oculus.com/blog/introducing-app-lab-a-new-way-to-distribute-oculus-quest-apps/),
+a way for developers to distribute apps directly to consumers safely and securely, via direct links
+or platforms like [SideQuest](https://sidequestvr.com/), without requiring store approval and
+without sideloading, which allows them to get an app directly to end users, even if it is early in
+development, experimental, or aimed at a unique audience.
+
+## Exemplary PWAs on the Oculus Quest&nbsp;2
+
+### PWAs by Meta
+
+Multiple Meta divisions have created PWAs for the Oculus Quest&nbsp;2, for example
+[Instagram](https://www.oculus.com/experiences/quest/6102857836422862) and
+[Facebook](https://www.oculus.com/experiences/quest/6126469507395223). These PWAs run in standalone
+app windows that do not have a URL bar and that can be freely resized.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/m5NoDaB7hyFOvrxHF9oS.jpeg", alt="Facebook Oculus Quest 2 app.", width="800", height="450" %}
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/gxYCh0Z9R3vXRU0MWsIB.jpeg", alt="Instagram Oculus Quest 2 app", width="800", height="450" %}
+
+### PWAs by other developers
+
+At the time of this writing, there are a couple of other developers that have published PWAs for the
+Oculus Quest&nbsp;2 on the Oculus Store.
+[Spike](https://www.oculus.com/experiences/quest/4949538568409451) lets users experience all the
+essential work tools like email, chat, calls, notes, tasks, and to-dos from their inbox in a virtual
+environment hub.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/yATTFGQRz75I2JrIAUSz.jpeg", alt="Spike Oculus Quest 2 app.", width="800", height="450" %}
+
+Another example is [Smartsheet](https://www.oculus.com/experiences/quest/4636949963023496), a
+dynamic workspace that empowers teams to manage projects, automate workflows, and rapidly build new
+solutions.
+
+More PWAs like Slack, Dropbox, or Canva are coming, as teased in a
+[video](https://www.facebook.com/watch/?v=4637130066326723) featuring Jacob Rossi that was released
+in the context of Facebook's Connect conference in 2021.
+
+## Creating PWAs for Oculus
+
+Meta has outlined the required steps in their
+[documentation](https://developer.oculus.com/documentation/web/pwa-gs/). In general, PWAs that are
+installable in Chrome should oftentimes work out of the box.
+
+### Web App Manifest requirements
+
+There are some important differences compared to
+[Chrome's installability criteria](https://web.dev/install-criteria/#criteria) and the
```

Comment:
```suggestion
[Chrome's installability criteria](/install-criteria/#criteria) and the
```

Reply:
We have permission to keep the Fugu stuff together until we have feature parity between the two sites.